libs/wallet/src/web3-react/utils/switchChain.ts

```diff
@@ -9,16 +9,11 @@ import { getIsWalletConnect } from '../hooks/useIsWalletConnect'
 import { getChainInfo, RPC_URLS } from '@cowprotocol/common-const'
 
 function getRpcUrls(chainId: SupportedChainId): [string] {
-  switch (chainId) {
```

Comment:
This why using switch is not type-safe.
I added a value for Sepolia here only when I got an error in runtime.
But with using `Record` I could get it on compile level

Reply:
`Record` all the way!